requirements.txt

```diff
@@ -1,18 +1,106 @@
-setuptools
-nipype
-SimpleITK
-scipy==1.5.4
-matplotlib==3.3.3
+absl-py==1.4.0
+astunparse==1.6.3
+attrs==23.1.0
+batchgenerators==0.25
+cachetools==5.3.1
+certifi==2023.5.7
+charset-normalizer==3.1.0
+ci-info==0.3.0
+click==8.1.3
+cloudpickle==2.2.1
+cycler==0.11.0
+decorator==4.4.2
+DeepMRSeg @ git+https://github.com/CBICA/DeepMRSeg.git@f351affff0d79a134fbc4073582f798efc81e409
+dicom2nifti==2.4.8
```

Comment:
Why is this needed? We are not handling any dicom data. Please remove if not needed.